File: Algorithm/Jungol/jungol_1291/jungol_1291.c
// http://www.jungol.co.kr/bbs/board.php?bo_table=pbank&wr_id=574&sca=2010

#include <stdio.h>

void printResult(int, int);

int main(){
    int s, e;
    while(1){
        scanf("%d %d", &s, &e);
        if(2<=s && s<=9 && 2<=e && e<=9) break;
        else {
            printf("INPUT ERROR!\n");
        }
    }
    printResult(s, e);

    return 0;
}

void printResult(int s, int e){
    int i, j;
    if(s<e){
        for(i=1; i<=9; i++){
            for(j=s; j<=e; j++){
                printf("%d * %d = %2d   ", j, i, j*i);
            }
            printf("\n");
        }
    }
    else{
        for(i=1; i<=9; i++){
            for(j=s; j>=e; j--){
                printf("%d * %d = %2d   ", j, i, j*i);
            }
            printf("\n");
        }
    }
}

File: Algorithm/Jungol/j1291/jungol_1291.py
# http://www.jungol.co.kr/bbs/board.php?bo_table=pbank&wr_id=574&sca=2010

def printResult(s, e):
    if s < e:
        for i in range(1, 10):
            for j in range(s, e+1):
                print('{0} * {1} = {2:2}'.format(j, i, j*i), end='   ')
            print(sep='\n')
        else:
            for i in range(1, 10):
                for j in range(s, e-1, -1):
                    print('{0} * {1} = {2:2}'.format(j, i, j*i), end='   ')
                print(sep='\n')


while(True):
    s, e = map(int, input().split())
    if s >= 2 and s <= 9 and e >= 2 and e <= 9:
        break
    else:
        print("INPUT ERROR!")

printResult(s, e)
